### ISSUE TITLE
Simulation does not stop after polyembryony is lost

### ISSUE THREAD

**Issue by PyhaTanja** [ybrandvain/polyembryony#7]
Simulation does not stop after polyembryony is lost even when I have 
`gen.after.loss = 15,  gen.after.fix = 15 `

**Comment by ybrandvain** [comment]
i think this works.. but you need n.gen to be small (less than gen.after.loss = 15 and gen.after.fix
